torchani/data/new.py

```diff
@@ -0,0 +1,449 @@
+import numpy as np
+import torch
+import functools
+from ._pyanitools import anidataloader
+import importlib
+pkbar_installed = importlib.util.find_spec('pkbar')
+if pkbar_installed is not None:
+    import pkbar
+
+
+class CacheDataset(torch.utils.data.Dataset):
+
+    def __init__(self, file_path,
+                 species_order='HCNO',
+                 transform=True,
+                 self_energies=[-0.600953, -38.08316, -54.707756, -75.194466],
+                 batch_size=1000,
+                 device='cpu',
+                 bar=20,
+                 test_bar_max=None):
+
+        super(CacheDataset, self).__init__()
+
+        # species_dict = {'H': 0, 'C': 1, 'N': 2, 'O': 3}
```

Comment:
Please remove unused code. Comments should be used to tell the code reader what this part of the code is doing, not to store dead code.

Reply:
it's telling how the species_dict should look like.
I changed it to
```py
        # example of species_dict will looks like
        # species_dict: {'H': 0, 'C': 1, 'N': 2, 'O': 3}
        # self_energies_dict: {'H': -0.600953, 'C': -38.08316, 'N': -54.707756, 'O': -75.194466}
```